src/sdk/namespace_fs.js

```diff
@@ -2152,6 +2225,25 @@ class NamespaceFS {
         };
     }
 
+    // We can have only one versioned object with null version ID per key.
+    // It can be latest version, old version in .version/ directory or delete marker
+    // This function removes an object or delete marker with a null version ID inside .version/ directory 
+    async _delete_null_version_from_versions_directory(key, fs_context, is_gpfs) {
```

Comment:
seems redundant to pass both fs_context and is_gpfs. just call is_gpfs inside with fs_context.